Add headings and links for each SEM in 2023  (Fixes #538)

davidstanke opened the pull request:
Fixes #538

Preview:
https://doradotdev-staging--pr542-drafts-on-di4mt0dy.web.app/research/2023/structural-equation-models/#2-documentation-is-foundational

dora-machine-user commented:
Cloud Build time: Tue Feb  6 22:16:28 UTC 2024
Preview this PR [with drafts on] at https://doradotdev-staging--pr542-drafts-on-di4mt0dy.web.app
Preview this PR [with drafts off] at https://doradotdev-staging--pr542-drafts-off-e4t5a183.web.app

davidstanke commented:
> Bit of a nit, but when you use an ordered list starting at 1, the numbers don't match the chapter numbers in the report.

Whoa, where did those numbers even come from? I mean, obviously, I put them there, but I can't figure out why--I thought I had copied them from the headings in the images but the evidence points to other conclusions. Anyway, I removed them! I'd rather not have any numbers, because in the context of the site, things aren't strongly structured according to report chapters.

And yeah, I removed the headers from the images. When I first added the headings, they were there, and duplicated, and it was hideous, so I cropped them out. You're right that they'll lack context if they're copied out, but I agree this is an okay tradeoff.

Thanks for the quick review!
